care/facility/migrations/0127_patientsearch_is_active.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.11 on 2020-06-26 10:39
```

Comment:
Codacy found an issue: [Similar lines in 2 files](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5765271)